### DISCUSSION TITLE
chromebook

### DISCUSSION THREAD

**Discussion by FalkoBabbage** [AppGround-io/bluetooth-keyboard-and-mouse-support#181] [Bugs & Problems]
Hi all,

I have tried to connect my Samsung tablet A of 2016 to two devices. One is a chromebook Asus Flip C101PA-FS002 with Chrome OS and the other is a Lenovo Thinkpad with Windows 10. Both connections did not work however.

My chromebook did not find the my tablet in the bluetooth devices list. It does have bluetooth 4.0 however.
My lenovo thinkpad did find the tablet but I could not connect to it. It first ask on the tablet as well as the laptop to confirm the connection but it just canceled it.

Can someone help?
Kind regards,
Falko

**Comment by zorazorr** [comment]
Your Samsung Tab A needs Android 9.

**Reply by FalkoBabbage** [reply]
Ah okay. It does work with my Nokia 1.3 and Levono Thinkpad but it still does not work with my chromebook.

Just to be clear: It says chrome OS is supported, but in what way? Like, can I control a computer with a chromebook via the app or can I control a chromebook with a mobile phone via the app?
